tests/xbot/common/injection/BaseWPITest.java

```diff
@@ -38,6 +39,9 @@ public void setUp() {
         RobotState.SetImplementation(mockRobotState);
 
         propertyManager = injector.getInstance(PropertyManager.class);
+        
+        SafeRobotAssertionManager assertionMan = injector.getInstance(SafeRobotAssertionManager.class);
+        assertionMan.setExceptionsEnabled(true);
```

Comment:
It would be cleaner to have this set as part of the UnitTestModule() at the guice level so people don't need to remember to do this themselves.